Sources/Swift/Integrations/SessionReplay/RRWeb/SentryRRWebBreadcrumbEvent.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+@objcMembers
+class SentryRRWebBreadcrumbEvent: SentryRRWebCustomEvent {
```

Comment:
`m`: At some point, we should add some tests for this class, but you can also do this in an upcoming PR.

Reply:
Yes, the tests are coming in the next PR where I will be using this class.